### PR TITLE
contrib/dnscrypt-proxy: new package (2.1.5)

### PR DESCRIPTION
**Pull request by flukeeey**

### CODE REVIEW

**Comment by q66** [comment]
ping

**Comment by flukeeey** [comment]
Oops, I had `check = no` set in config.ini, so missed this. Fixed.

File: contrib/dnscrypt-proxy/files/dnscrypt-proxy [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# dnscrypt-proxy service
+
+type            = process
+command         = /usr/bin/dnscrypt-proxy -config /etc/dnscrypt-proxy/dnscrypt-proxy.toml
+logfile         = /var/log/dnscrypt-proxy/dnscrypt-proxy.log
```
Comment (q66) [inline contrib/dnscrypt-proxy/files/dnscrypt-proxy:5]: is there any reason to put the log in a subdir?

**Comment by q66** [comment]
besides the service file thing, probably lgtm

File: contrib/dnscrypt-proxy/files/dnscrypt-proxy (hunk 1 above)
Comment (flukeeey) [reply contrib/dnscrypt-proxy/files/dnscrypt-proxy:5]: dnscrypt-proxy has separate-file logging for handled DNS queries, suspicious domains, etc. Upstream (and other distros) group them together in a dnscrypt-proxy folder as per the default config.

**Comment by flukeeey** [comment]
Sweet, if you're okay with the log file explanation above, I'll go ahead and merge. :+1:

File: contrib/dnscrypt-proxy/files/dnscrypt-proxy (hunk 1 above)
Comment (q66) [reply contrib/dnscrypt-proxy/files/dnscrypt-proxy:5]: ah, understood

**Pull request merged**